setup_all.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source config_vars.sh
+
+set -e
+
+# Start MySQL server
+./run_mysql.sh
+
+# Build and run the API
+docker build -t zeeguu-api-core -f docker-files/zeeguu-api-core/Dockerfile .
+docker run --net=host -d \
+    -e MICROSOFT_TRANSLATE_API_KEY="$MICROSOFT_TRANSLATE_API_KEY" \
+    -e GOOGLE_TRANSLATE_API_KEY="$GOOGLE_TRANSLATE_API_KEY" \
+    -e WORDNIK_API_KEY='$WORDNIK_API_KEY' \
+    --name=zeeguu-api-core zeeguu-api-core
+
+# Build and run the Web UI
+docker build -t zeeguu-web --build-arg ZEEGUU_API__EXTERNAL="http://$SERVER_IP:9001" -f docker-files/zeeguu-web/Dockerfile .
```

Comment:
is it ok that the 9001 is hardcoded here?

Reply:
the apache conf actually expects the api to be on port 8080 at the moment.

Reply:
It should be http://<server_ip>/api at the moment. The apache from Zeeguu-Web will forward requests to Zeeguu-API.